### PR TITLE
front: add webkit browser to e2e tests

### LINKED ISSUE #8645: front: Add webkit for e2e tests
# Description and goal

Add webkit for e2e test for them to work on Safari.

### Acceptance criteria

- [x] All e2e tests work on WebKit

### PR DESCRIPTION
**Pull request by Maymanaf**
Closes #8645

### CODE REVIEW

File: front/playwright.config.ts [hunk 1]
```diff
@@ -63,5 +63,15 @@ export default defineConfig({
...
+    {
+      name: 'webkit',
+      use: {
+        browserName: 'webkit',
+        launchOptions: {
+          slowMo: 500, // Slows down WebKit interactions by 500 milliseconds
+        },
+      },
+      dependencies: ['setup'],
+    },
```
Comment (Yohh) [inline front/playwright.config.ts:75]: shouldn't we only use test.slow() for specifics tests? I'm afraid that the actual delay could hide some performance issues

**Review by Yohh — COMMENTED** [review]
nice! I'm glad to finally see it!

File: front/playwright.config.ts (hunk 1 above)
Comment (Maymanaf) [reply front/playwright.config.ts:75]: WebKit is around 1.7 times slower than Firefox and Chromium, according to feedback from the Playwright community. Therefore, adding `slowMo` ( only 500ms) limits the number of tests on which we apply `test.slow()`.

**Review by Yohh — APPROVED** [review]
LGTM

we should be carefull, tests are a lot flaky on macOs if others apps are open

**Review by SharglutDev — APPROVED** [review]
A lot of tests are failing for me too in local (macOS)

**Comment by Maymanaf** [comment]
> A lot of tests are failing for me too in local (macOS)

@SharglutDev @Yohh Locally, it's much slower than on the CI, and the application's performance on an unloaded infrastructure is not optimal. That's why it's better to limit the number of workers and increase the timeout. I tested several builds and solutions on the CI before submitting the PR, and it works well now. Only the 003 tests remain unstable, but they've been giving this result for a while. I'm going to have to modify the logic behind them

**Comment by SharglutDev** [comment]
Maybe we should add a readme somewhere or complete the osrd documentation to explain the best way to launch the e2e tests in local.

**Comment by Yohh** [comment]
> Maybe we should add a readme somewhere or complete the osrd documentation to explain the best way to launch the e2e tests in local.

or in this specific case, we should limit the workers to 1 by default, continue to write tests parallelize and let the user increase the workers if he wants

if the tests are too huge to work parallelized we shouldn't let the workers to 50%

**Comment by SharglutDev** [comment]
If we do that, the tests will take longer on the CI or it can be set only for local env ?

**Comment by Yohh** [comment]
yes we can keep 50% for CI, 1 for local

**Pull request merged**